src/components/user-profile.component.js

```diff
@@ -141,6 +143,15 @@ export const UserProfile = ({
         </Text>
       </TouchableOpacity>
 
+      <TouchableOpacity style={styles.unit}>
+        <Text style={styles.unitNumber}>
+          {starCount}
+        </Text>
+        <Text style={styles.unitText}>
+          {translate('common.star', language)}
```

Comment:
It is better to use the plural (`stars`), by analogy with other (`repositories`, `followers`).